wrapper/CSharp/wolfSSL_CSharp/wolfSSL.cs

```diff
@@ -1084,6 +1128,75 @@ public static void CTX_free(IntPtr ctx)
             }
         }
 
+        public static void CTX_set_servername_callback(IntPtr ctx, sni_delegate sni_cb) 

```

Comment:
Is it possible to add these callbacks to the TLS examples so this new code is tested? Even if it just registers a callback that does a stdout print?